tar_binary_restarts/tar_binary_restarts.py

```diff
@@ -1,22 +1,19 @@
 import os
-import yaml
-import os
 import subprocess
 import time
-import pdb
 from esm_parser import user_note
 
 def tar_binary_restarts(config, test_config=None):
     """
-    Tar binary restart files.
+    Tar and compress binary restart files.
```

Comment:
Can you add here that this is specific to FESOM-2?

Reply:
Done